.github/workflows/security-scans.yml

```diff
@@ -0,0 +1,117 @@
+name: "Security Scans"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 1' # run tests at 1 AM (UTC), every monday (1)
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GO_VERSION: "~1.18"
+
+jobs:
+  prepare-security-scans:
+    name: "Prepare Security Scans"
+    runs-on: ubuntu-20.04
+    env:
+      RENDERED_CHART_FILENAME: "scan-tpl.yml"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: 'true'
+
+      - name: Render helm template
+        run: |
+          VERSION=$(cat './VERSION.txt')
+          cd ./installer/manifests/keptn
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+          helm dependency update ./
+          helm dependency build ./
+          # replace "appVersion: latest" with "appVersion: $VERSION" in all Chart.yaml files
+          find . -name Chart.yaml -exec sed -i -- "s/appVersion: latest/appVersion: ${VERSION}/g" {} \;
+          helm template ./ -n keptn --name-template keptn --version 0.17.0 > ${{ env.RENDERED_CHART_FILENAME }}
```

Comment:
would it make sense to get the version from the latest semver tag?

Reply:
yes, i will fix that :)